lib/order.rb

```diff
@@ -0,0 +1,107 @@
+require 'csv'
+require 'awesome_print'
+require_relative 'customer'
+
+class Order
+  
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+  
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    
+    @id = id
+    @products = products
+    @customer = customer
+    @fulfillment_status = fulfillment_status
+    
+    valid_statuses = [:pending, :paid, :processing, :shipped, :complete]
+    
+    if valid_statuses.include?(fulfillment_status)
+      return fulfillment_status
+    else
+      raise ArgumentError
```

Comment:
You should include a message with your `ArgumentError`s so the user knows what is wrong.

```suggestion
      raise ArgumentError.new("Invalid fulfillment status: #{fulfillment status}")
```